frontend/src/app/modules/main/playlist/artist-list/artist-list.component.ts

```diff
@@ -12,9 +14,13 @@ export class ArtistListComponent implements OnInit {
   userId: number = 1; // Temporary value
   artists!: ArtistReadDTO[];
 
-  constructor(
-    private _reactionService: ReactionService
-  ) { }
+  constructor(private _reactionService: ReactionService, private _authService: AuthService) {
+    this._authService.getAuthStateObservable()
+      .pipe(filter((state) => state !== null))
```

Comment:
You could simplify it
.pipe(filter((state) => state !== null)) -> .pipe(filter((state) => state))